src/HTTPure/Utils.purs

```diff
@@ -1,23 +1,20 @@
 module HTTPure.Utils
-  ( encodeURIComponent
-  , decodeURIComponent
+  ( module Global
+  , encodeURIComponent
```

Comment:
I just realized that nothing is using `decodeURIComponent` any more outside this module so there's no need to export it.  There's also only one use of `encodeURIComponent` (https://github.com/cprussin/purescript-httpure/blob/master/src/HTTPure/Request.purs#L43) so we don't need to export that either and could just update that call site instead.

No need to make those changes, just wanted to note it somewhere since I just realized after merging the PR.